config/software/curl.rb

```diff
@@ -30,7 +30,7 @@
 
   if freebsd?
     # from freebsd ports - IPv6 Hostcheck patch
-    patch source: "curl-freebsd-hostcheck.patch", plevel: 1
+    patch source: "curl-freebsd-hostcheck.patch", plevel: 1, env: env
```

Comment:
I agree - having patch, bundle, rake, make etc. all use env should indeed be the default.  Unfortunately, I don't know how to change ruby DSL methods to look for env as defined in the builder or software definitions.  Maybe introducing an env method in builder would be the way to go?  In any case - the blanket addition of env was mechanical and hasn't otherwise affected anything else.  I'll revisit it later if I continue playing with omnibus.